src/main/java/QueryServlet.java

```diff
@@ -134,24 +134,49 @@ private String sendError(String errorMessage) {
     return jsonResponse.toString();
   }
 
+  private String getLocation(
```

Comment:
Could you update the name of this method to make it more clear what it does? In particular, it's confusing to me that getLocation requires an input parameter of location.

Reply:
A comment above the method would be great too actually

Reply:
Done!